src/utils/checkFormat.js

```diff
@@ -3,5 +3,5 @@ import IsHex from '../utils/isHex';
 export default (token) => {
 	// This util will check the basic format of the token.
 
-	return IsHex(token);
+	return token.length !== 16 || IsHex(token);
```

Comment:
@seanmcneilly2  Doesn't this need to be ```=== ``` instead? Assuming that checkFormat returns true for valid token.